src/main/java/com/google/sps/servlets/RatingServlet.java

```diff
@@ -44,10 +64,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         int rating = Integer.parseInt(request.getParameter("rating"));
 
         // Update tutor's rating
-        SampleData.rateTutorByEmail(tutorEmail, studentEmail, rating);
+        datastore.rateTutor(tutorEmail, studentEmail, rating);
 
-        String jsonTutors = new Gson().toJson(SampleData.getSampleTutors());
-        String jsonStudents = new Gson().toJson(SampleData.getSampleStudents());
+        String jsonTutors = new Gson().toJson(datastore.getTutors());
+        String jsonStudents = new Gson().toJson(datastore.getStudents());
 
         String json = new Gson().toJson(new String[]{jsonTutors, jsonStudents}); 
```

Comment:
I ended up just removing these lines in my PR because I don't think we use the list of tutors and students returned anywhere.

Reply:
They are being used inside RatingTest.java to check the if the returned contents have the new ratings that they should have. I think that if we remove these lines the tests would fail. Do you know a better way we could use to verify if the mock database was updated as it should have been? I have been just passing it down as a json to do that.

Reply:
Oh, I think the tests worked for me without these lines because we weren't using the json to test. I think we were just using to the TutorSession object to check if its properties changed.